<launch>
  <arg name="red" default="spectral_670"/>
  <arg name="nir" default="spectral_800"/>
  <arg name="bag_start" default="0"/>
  <arg name="bag_rate" default="1"/>

  <node pkg="rosbag" type="play" name="player" output="screen"
    args="-s $(arg bag_start)
          -r $(arg bag_rate)
          /home/chao/Workspace/bag/ndvi_row_23_34.bag"/>

  <!-- image_proc -->
  <node pkg="image_proc" type="image_proc" name="image_proc" ns="$(arg red)"/>
  <node pkg="image_proc" type="image_proc" name="image_proc" ns="$(arg nir)"/>

  <!-- rosbag record -->
  <node pkg="rosbag" type="record" name="recorder" output="screen" args="-o $(arg tag)
    /$(arg red)/camera_info
    /$(arg red)/image_rect
    /$(arg nir)/camera_info
    /$(arg nir)/image_rect
    "/>

</launch>
